### ISSUE TITLE
ClassName is not working with notification system

### ISSUE THREAD

**Issue by Jonyberry** [mantinedev/mantine#979]
### What package has an issue

@mantine/notifications

### Describe the bug

When I try to use className prop in `showNotification` function, class won't appear in DOM.

```
notifications.showNotification({
  message: 'Leave the building immediately',
  className: 'my-notification-class',
});
```
This code shows notification but without class `my-notification-class`.

### In which browser did the problem occur

Chrome

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/mantine-notification-class-not-showing-jvcedy?file=/src/App.js

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by Jonyberry** [comment]
I found out, that in `mantine-notifications/src/NotificationContainer/NotificationContainer.tsx` is being ignored `className` prop inside `notification` object. I think it just need to be changed from
```
<Notification
      sx={sx}
      className={className}
      ...
    >
      {notification.message}
 </Notification>
```
to
```
<Notification
      sx={sx}
      className={cx(className, notification.className)}
      ...
    >
      {notification.message}
 </Notification>
```
or something like that.
If I'm right, I can fix it.

**Comment by Jonyberry** [comment]
Please can you tell me if this is a real problem or am I missing something?

**Comment by rtivital** [comment]
It was fixed in dev branch and will be available with 4.1 release.

**Comment by Jonyberry** [comment]
Thanks a lot !!
